sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2024-2025.bpmn

```diff
@@ -72,1352 +216,1474 @@ dmnFullTimeX (comes from dmnFullTime)</bpmn:text>
+      <bpmn:outgoing>Flow_1rfinbu</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0a9extn" name="calculatedDataPartnerStudentStudyWeeks">
+    <bpmn:intermediateThrowEvent id="Event_1bun4vx" name="Total Second Residence">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if studentDataPartnerStudyWeeks = null then 0 else studentDataPartnerStudyWeeks" target="calculatedDataPartnerStudentStudyWeeks" />
+          <zeebe:output source="=if&#10;  studentDataLivingWithPartner = &#34;no&#34; and studentDataRelationshipStatus = &#34;married&#34; and&#10;  offeringDelivery = &#34;onsite&#34;&#10;then&#10;dmnFullTimeCalculatedDataProgramYearMaximums.limitWeeklySecondResidence * offeringWeeks&#10;else&#10;  0" target="calculatedDataTotalSecondResidence" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0n7won7</bpmn:incoming>
-      <bpmn:outgoing>Flow_10kp7wy</bpmn:outgoing>
+      <bpmn:incoming>Flow_1rfinbu</bpmn:incoming>
+      <bpmn:outgoing>Flow_033myat</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0bk5jrr" name="calculatedDataPartnerEmploymentInsturanceBenefits">
+    <bpmn:exclusiveGateway id="Gateway_0e92cz7" name="Additional Transportation" default="Flow_1lx5dkf">
+      <bpmn:incoming>Flow_1j90pyg</bpmn:incoming>
+      <bpmn:outgoing>Flow_1lx5dkf</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0incm38</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_1bl1yut" name="Additional Transportation">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=partner1EmploymentInsuranceBenefits" target="calculatedDataPartnerEmploymentInsturanceBenefits" />
+          <zeebe:output source="=0" target="calculatedDataTotalAdditionalTransportation" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_02m6u8j</bpmn:incoming>
-      <bpmn:outgoing>Flow_0l4ido8</bpmn:outgoing>
+      <bpmn:incoming>Flow_1lx5dkf</bpmn:incoming>
+      <bpmn:outgoing>Flow_0bu4f9h</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_1wpkfdj" name="calculatedDataPartnerEmploymentInsturanceBenefits">
+    <bpmn:businessRuleTask id="Activity_01s5l73" name="Return Transportation">
       <bpmn:extensionElements>
+        <zeebe:calledDecision decisionId="dmnFullTimeApplicationTransportationMaximum" resultVariable="dmnFullTimeCalculatedDataApplicationtransportationMaximum" />
         <zeebe:ioMapping>
-          <zeebe:output source="=studentDataPartnerEmploymentInsurance" target="calculatedDataPartnerEmploymentInsturanceBenefits" />
+          <zeebe:output source="=min(studentDataTransportationCost, dmnFullTimeCalculatedDataApplicationtransportationMaximum.limitApplicationTransportation)&#10;" target="calculatedDataReturnTransportationCost" />
```

Comment:
As discussed, When a dmn has only one output property, then it returns primitive value instead of an object as output. 

E.g 

## Primitive output
`dmnFullTimeApplicationTransportationMaximum(dmnFullTimeCalculatedDataApplicationtransportationMaximum)` 

![image](https://github.com/user-attachments/assets/37fb66f3-004d-4a4e-b0be-7867dac90cbb)

![image](https://github.com/user-attachments/assets/eef44239-f081-46ac-93ab-d2ff96520067)

## Object output
`dmnFullTimeCalculatedDataIncomeThreshold`

![image](https://github.com/user-attachments/assets/9ab47f0a-cbd4-45bd-8620-d573fee4e851)